src/components/wallet/earn/Validate/ValidatorInfo.vue

```diff
@@ -0,0 +1,207 @@
+                </span>
+            </div>
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import 'reflect-metadata'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import Spinner from '@/components/misc/Spinner.vue'
+import moment from 'moment'
+import { ava } from '@/AVA'
+import { BN } from '@c4tplatform/caminojs'
+import AvaxInput from '@/components/misc/AvaxInput.vue'
+import Tooltip from '@/components/misc/Tooltip.vue'
+
+@Component({
+    name: 'validator_info',
+    components: {
+        Spinner,
+        AvaxInput,
+        Tooltip,
+    },
+})
+export default class ValidatorInfo extends Vue {
+    @Prop() nodeId!: string
+    @Prop() nodeInfo!: any
+
+    startTime: any
+    endTime: any
+    upTime: any
+    reaminingValidation: any
+    depositAmount: any
+    txID: any
+
+    loading: boolean = true
+
+    mounted() {
+        this.getInformationValidator()
+    }
+
+    get maxAmt(): BN {
+        return ava.getNetwork().P.minStake
+    }
+
+    getInformationValidator() {
+        try {
+            this.loading = true
+            let today = moment()
+
+            this.startTime = moment(new Date(parseInt(this.nodeInfo.startTime) * 1000)).format(
+                'MMMM Do YYYY, h:mm:ss a'
+            )
+            this.endTime = moment(new Date(parseInt(this.nodeInfo.endTime) * 1000)).format(
+                'MMMM Do YYYY, h:mm:ss a'
+            )
+            this.upTime = parseFloat(this.nodeInfo.uptime) * 100
+
+            var reaminingValidationDuration = moment.duration(
+                today.diff(moment(new Date(parseInt(this.nodeInfo.startTime) * 1000)))
+            )
+
+            let dataReaminingValdiationDuration = {
+                days: reaminingValidationDuration.days().toString(),
+                hours:
+                    reaminingValidationDuration.hours() > 9
+                        ? reaminingValidationDuration.hours().toString()
+                        : `0${reaminingValidationDuration.hours().toString()}`,
+                minutes:
+                    reaminingValidationDuration.minutes() > 9
+                        ? reaminingValidationDuration.minutes().toString()
+                        : `0${reaminingValidationDuration.minutes().toString()}`,
+                seconds:
+                    reaminingValidationDuration.seconds() > 9
+                        ? reaminingValidationDuration.seconds().toString()
+                        : `0${reaminingValidationDuration.seconds().toString()}`,
+            }
+
+            let strRemainingValidation = `${dataReaminingValdiationDuration.days} Days ${dataReaminingValdiationDuration.hours}h ${dataReaminingValdiationDuration.minutes}m ${dataReaminingValdiationDuration.seconds}s`
+            this.reaminingValidation = strRemainingValidation
+            this.depositAmount = parseFloat(this.nodeInfo.stakeAmount) / 1000000000
+            this.txID = this.nodeInfo.txID
+            this.loading = false
```

Comment:
Please move `this.loading = false` to `finally` block, as this will never happen if an error occurs in this `try` block, resulting in infinite `loading` state